lib/fizzy/execute.cpp

```diff
@@ -1375,6 +1390,42 @@ ExecutionResult execute(Instance& instance, FuncIdx func_idx, span<const Value>
             stack.push(static_cast<uint32_t>(stack.pop()));
             break;
         }
+        case Instr::i32_trunc_f32_s:
+        {
+            if (!trunc<float, int32_t>(stack))
+            {
+                trap = true;
```

Comment:
Do we trap instead of undefined result? Why? This seems against the spec.

Reply:
I updated the description with the explanation. And spec tests confirm that.